include/libopencm3/stm32/f0/adc.h

```diff
@@ -190,8 +190,8 @@ void adc_calibrate_wait_finish(uint32_t adc)
 void adc_enable_analog_watchdog_on_all_channels(uint32_t adc);
 void adc_enable_analog_watchdog_on_selected_channel(uint32_t adc, uint8_t chan);
 void adc_disable_analog_watchdog(uint32_t adc);
-void adc_set_watchdog_high_threshold(uint32_t adc, uint8_t threshold);
-void adc_set_watchdog_low_threshold(uint32_t adc, uint8_t threshold);
+void adc_set_watchdog_high_threshold(uint32_t adc, uint16_t threshold);
+void adc_set_watchdog_low_threshold(uint32_t adc, uint16_t threshold);
```

Comment:
+1 - and indeed, same stuff for F3 ADC_TR1..

Reply:
done